torch/autograd/gradcheck.py

```diff
@@ -1941,8 +1946,9 @@ def gradcheck(
     fast_mode: bool = False,
     masked: Optional[bool] = None,
 ) -> bool:
-    r"""Check gradients computed via small finite differences against analytical
-    gradients wrt tensors in :attr:`inputs` that are of floating point or complex type
+    r"""Check gradients computed via small finite differences against analytical gradients wrt given tensors.
```

Comment:
I prefer the original wording here.

Reply:
Alright. I have done the original wording here. Please review this.
@soulitzer